src/calendar/CalendarUpdateDistributor.js

```diff
@@ -72,6 +73,21 @@ export class CalendarMailDistributor implements CalendarUpdateDistributor {
 			body: makeInviteEmailBody(sender, event, message),
 			event,
 			sender
+		}).catch(RecipientsNotFoundError, e => {
+			// we want to delete the event even if the recipient is not an existing tutanota address
+			// and just exclude them from sending out updates but leave the event untouched for other recipients
+			const invalidRecipients = e.message.split("\n")
```

Comment:
can we be sure that the message will always have the expected format?

Reply:
I would imagine just something like this:

```
export class RecipientsNotFoundError extends TutanotaError {

	+recipients: Array<string>

	constructor(recipients: Array<string>) {
		super("RecipientsNotFoundError",  recipients.join("\n"))
	}
}
```
 

> I am not sure we want to create this one error in a different way than all others.

I think it would be fine to do, we just haven't done it yet because we haven't needed to

Reply:
Yeah, I just looked into it a bit more and I am still not convinced we want to make such a change. The reason is that currently we pass errors between the worker and main thread in messages and we would have to handle this error differently from all others then, because we cannot simply pass the message as it is the only one where we do not pass a string to the constructor. Take a look at: `objToError()` which is what we use to recreate the error after we receive such a message from the worker.

Reply:
So, I pushed another commit, but it does not work of course until we handle the beforementioned problem

Reply:
Ah I see, yeah that complicates things. Better to leave it how it is then, making it work nicely would probably be a bigger change than it's worth